### DISCUSSION TITLE
Is there anyway to make Email ID optional and use Mobile Number instead in user store?

### DISCUSSION THREAD

**Discussion by gunpal5** [abpframework/abp#9377] [Q&A] [answered]
Hello,

I want to use ABP Framework for backend APIs for a mobile app. We don't need Email ID, we want to make email id optional. Is there any way to do this with ABP Framework?

Regards,
Gunpal Jain

**Comment by realLiangshiwei** [comment, marked as answer]
Try:

```csharp
Configure<IdentityOptions>(options=>
{
    options.User.RequireUniqueEmail = false;
});
```

**Reply by gunpal5** [reply]
Thank you for your help.
